ssc_interface_wrapper/launch/ssc_interface_wrapper.launch

```diff
@@ -17,6 +17,33 @@
 	This file is used to launch a ros wrapper which can be combined with ssc_interface and pacmod drivers to make them CARMA compatible
 -->
 <launch>
+  <!-- Flag to toggle carma_record-->
+  <arg name="enable_ros1_record" default="false" doc="Flag to toggle carma_record"/>
+
+  <!-- Directory of vehicle config parameters -->
```

Comment:
Its not clear to me why the vehicle_calibration_dir and VehicleConfigParams are being loaded, when they were not loading before.
I mean, are they needed for ros1 record launch?

Reply:
Calibration is not needed. Config is needed to load UniqueVehicleParams which has the topic exclusion rules for carma_record.